.circleci/continue_config.yml

```diff
@@ -624,43 +251,6 @@ workflows:
           filters:
             branches:
               only: main
```

Comment:
I am not sure on what this is doing, @merelcht do you have more context on this? Is this something we need to move to GitHub actions or is it safe to remove?